### PR TITLE
Remove Message prefix

### PR DESCRIPTION
**Pull request by jiivan**
realted to: golemfactory/golem-messages#20

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -47,4 +47,4 @@ numpy==1.13.1
 scipy==0.19.1
 pyssim
 enforce==0.3.4
--e git://github.com/golemfactory/golem-messages.git@v1.1.1#egg=golem_messages
+-e git://github.com/golemfactory/golem-messages.git@20_msgprefix#egg=golem_messages
```
Comment (Krigpl) [inline requirements.txt]: Why won't make you a release instead of this custom `20_msgprefix` tag?

Comment (jiivan) [reply requirements.txt]: I want to make a release when this PR is accepted, and update requirements before merging.

Comment (Krigpl) [reply requirements.txt]: What stops you from making the release before this PR is accepted?

Comment (jiivan) [reply requirements.txt]: Because it would stop every other PR related to golem-messages

**Review by Krigpl — APPROVED** [review]

**Pull request merged**